src/Auth0.ManagementApi/Clients/ClientGrantsClient.cs

```diff
@@ -77,11 +77,7 @@ public Task<IPagedList<ClientGrant>> GetAllAsync(GetClientGrantsRequest request,
         /// <returns>The <see cref="ClientGrant"/> that has been updated.</returns>
         public Task<ClientGrant> UpdateAsync(string id, ClientGrantUpdateRequest request)
         {
-            return Connection.PatchAsync<ClientGrant>("client-grants/{id}", request, 
-                new Dictionary<string, string>
-                {
-                    {"id", id}
-                });
+            return Connection.SendAsync<ClientGrant>(new HttpMethod("PATCH"), BuildUri($"client-grants/{id}"), request);
```

Comment:
Why not `HttpMethod.Patch`?

Reply:
Annoyingly does not exist in the .NET framework :(